app/views/metadata_presenter/analytics/_google.html.erb

```diff
@@ -0,0 +1,12 @@
+<% analytics.each do |analytic| %>
+  <% if ENV[analytic].present? %>
+    <script async src="<%= "https://www.googletagmanager.com/gtag/js?id=#{ENV[analytic]}" %>"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', <%= ENV[analytic] %>);
+    </script>
+  <% end %>
+<% end %>
```

Comment:
Pretty sure there are different snippets required for different Analytics services (e.g. depending on the account type in use).

I copied some examples down in this ticket https://trello.com/c/nJgqBCn1/2565-google-analytics-kick-off
(see comments)

Is this just a starting point or meant to be something that covers the different types of Analytics available from Google?

Reply:
i thought there were entirely different snippets as well, but having now gone to the analytics dashboard to generate the snippet for the three different types they all use the same script tags 🤷‍♀️